### DISCUSSION TITLE
What is the motivation for retrospective funders to fund impact evaluations?

### DISCUSSION THREAD

**Discussion by ccerv1** [hypercerts-org/hypercerts#530] [Q&A]
This question came in via Telegram from a member of our community. Here is the response I provided:

---

The primary motivation for a funder to fund impact evaluations is to separate the interests of the projects and the evaluators, because there could be some conflict. 

We expect most **projects** will have incentives to "highlight only positive impacts about their project" and "try to attract the most funding to their project". This makes it difficult for them to be objective about measuring their own impact. By contrast, an independent **impact evaluator**, who is seeing multiple projects, should be in a better position to remain neutral. Ideally, they are motivated by an incentive such as "develop a reputation as a credible impact evaluator". 

The ideal **funder** profile is someone who cares about impact and wants to fund projects that have more credible impact claims. We hope they are motivated by a desire to fund projects that have high quality impact evaluations from independent, high reputation impact evaluators. 

Ideally, a funder or group of funders is able to set up an impact evaluation pool, eg, they allocate 5% of funding to reward impact evaluations for ALL projects they are considering, not just the good ones. Then there is an incentive to evaluate both good and bad projects. And the evaluator gets paid for all their work, not just evaluating the best work.

See also these resources:
- [Flowchart](https://hypercerts.org/docs/whitepaper/ifs/) from the hypercerts whitepaper (specifically, Step 5)
- [Example](https://mirror.xyz/hypercerts.eth/kpu2vewrOVCCVNFf78Y1ui0D091XS6-ZgPoEKc0rkp0) of a retro pool with fixed amount allocated for impact evalaution

**Comment by ryscheng** [comment]
Here's the short answer I shared with Tereza

I suspect that impact evaluations will be critical in helping understand the validity and verify a project’s impact. Sometimes the funder may have in-house expertise to evaluate a Hypercert, but often times funders may need to outsource and pay for this expertise. This is certainly true in carbon markets, where they call it MRV. At the end of the day funders are motivated to deploy funding to do good, but may need help understanding the good from the bad.

**Comment by marsrobertson** [comment]
> independent impact evaluator

...who is getting data from where?

> highlight only positive impacts about their project

...my theory is that whichever data is available it will come from the project itself.

At the same time - availability of data is changing - satellites, drones, IoT sensors

I will not travel to Kenya to verify impact but I can totally pay $10 to someone local to take some pictures, record a short video, take a few samples to the lab.

I genuinely believe it is possible to manufacture $10 device that you plug to 3.5mm headphone jack to sample the soil, run on-device AI analysis, then send the results with timestamp / geolocation / POAP.

---

> funders may need to outsource and pay for this expertise

> certainly true in carbon markets, where they call it MRV

I'm wondering what is your practical expertise in this matter? See this [comment about Verra](https://github.com/hypercerts-org/hypercerts/discussions/449#discussioncomment-5327652).

I did the course here - https://bootup.airminers.org/ - 3x COP - IPCC reviewer - I seriously doubt that Hypercerts want to do the same "business as usual". Homework: try to onboard a project. Report back time and cost involved.

<img width="759" alt="image" src="https://user-images.githubusercontent.com/44369284/233774938-2e4cfe3a-f819-476b-b1bf-f41ccf72f8d6.png">


What I suggest instead:
* https://twitter.com/MRVProtocol
* https://mrv.earth/

### Other discussion
- https://github.com/hypercerts-org/hypercerts/discussions/449
- https://github.com/hypercerts-org/hypercerts/issues/439
- https://community.supermodular.xyz/t/hypercerts-retroactive-impact-funding-workflow-of-evaluation-incentives/380/14
